apps/admin-gui/src/app/shared/side-menu/side-menu-router.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { take } from 'rxjs/operators';
+import { ActivatedRoute, QueryParamsHandling, Router } from '@angular/router';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SideMenuRouterService {
+  constructor(private router: Router, private route: ActivatedRoute) {}
+
+  navigate(url: string[]): void {
+    this.route.queryParams.pipe(take(1)).subscribe((params) => {
+      let paramsHandlingMethod: QueryParamsHandling = 'merge';
+      const queryParams = Object.assign({}, params);
+
+      if (location.pathname.endsWith('applicationForm/preview')) {
+        paramsHandlingMethod = '';
+        delete queryParams.applicationFormItems;
+      }
```

Comment:
The problem here is that we can navigate await from the preview page by different means than just side menu e.g. links on vo detail. And different forms preserve the application form items params.

Reply:
I hope this should fix the problem with another type of navigation from the preview page (fixed for home button, on vo application preview for the vo link and the link for member/parent vo, on the group application preview for the group link and vo link).